src/tally.F90

```diff
@@ -441,6 +441,69 @@ subroutine score_general_ce(p, t, start_index, filter_index, i_nuclide, &
         end if
 
 
+      case (SCORE_PROMPT_NU_FISSION)
```

Comment:
On its own this looks fine.  But it does make me wonder if this can be combined with the delayed version.  It looks like the differences would be in the EMISSION_PROMPT constant and the call to score_fission_prompt_eout.  I think socre_fission_prompt_eout can be generalized to take an argument for the delayed group or prompt (i.e., 0 is prompt, a number > 0 is the group).  What do you think?